tests/testthat/test-NCBI_snp_query.R

```diff
@@ -53,13 +75,13 @@ test_that("ncbi_snp_query for rs146107628 (duplication)", {
   expect_equal(aa$Chromosome, "10")
   expect_equal(aa$BP, 98243085) ## on GRCh38
   expect_equal(aa$Marker, "rs146107628")
-  expect_equal(aa$Class, "indel")
+  expect_equal(aa$Class, "delins")
   expect_equal(aa$Gene, "R3HCC1L")
-  expect_equal(aa$Alleles, "T")
-  expect_equal(aa$Major, "T")
-  expect_equal(aa$Minor, "TT")
-  expect_equal(aa$MAF, 0.0365)
+  expect_equal(aa$Alleles, "T,TT")
   expect_equal(aa$AncestralAllele, "T")
+  expect_equal(aa$VariationAllele, "TT")
+  expect_equal(aa$MAF, 0.0365) 
+  expect_equal(aa$Minor, "T") 
```

Comment:
Hi @sinarueeger,
I've just been reviewing the output from the functions (and haven't looked at the code yet in detail). Here I see that the test has been updated so that the minor allele is  now "T", whereas before it was the duplication of "TT" (but I think the same variant is being considered as the Gnomad frequency has not changed). Has there been a shift in how this is represented by NCBI?